packages/gatsby/src/redux/actions/public.js

```diff
@@ -102,6 +102,11 @@ type ActionOptions = {
   followsSpan: ?Object,
 }
 
+type PageData = {
+  id: string,
+  result: ?Object,
```

Comment:
```suggestion
  resultHash: string,
```

Also adjust jsdocs below